src/main/update.ts

```diff
@@ -190,35 +190,36 @@ export async function update(update: Update, listeners: IUpdateOptions) {
   console.log("Finished to download:", dlPath);
 
   const extractPath =
-    process.platform == "darwin" // .app으로 실행한 건지 npm run dev로 실행한 건지도 확인해야 함
+    process.platform == "darwin" // we should check whether it was executed from .app or from npm run dev.
```

Comment:
A GFI candidate. Also change to yarn dev.
```suggestion
    process.platform == "darwin" // we should check whether it was executed from .app or from yarn dev.
```